test-three/side-menu-app/src/components/SimpleMenuItem.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+// icons
+import SettingsOutlinedIcon from '@material-ui/icons/SettingsOutlined';
+import AssessmentOutlinedIcon from '@material-ui/icons/AssessmentOutlined';
+import BrushOutlinedIcon from '@material-ui/icons/BrushOutlined';
+import TocOutlinedIcon from '@material-ui/icons/TocOutlined';
+import '../styles/style.css';
+
+export default function SimpleMenuItem({ name, subItems }) {
+
+  return ( 
+    <>
+      <ListItem button style={{ paddingLeft: 18 }}>
```

Comment:
Noticed a bit of styling baked in here. Is there a reason for this?

Reply:
For this I already had a css class. I changed the inline style in 'CollapsableMenuItem.js' where similar style was used but must have forgotten it in here ('SimpleMenuItem.js'). My bad. I have fixed this.